src/content/docs/apis/nerdgraph/examples/async-queries-nrql-tutorial.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "NerdGraph tutorial: Run NRQL queries asynchronously"
+tags:
+  - APIs
+  - NerdGraph
+  - Examples
+metaDescription: Use New Relic NerdGraph to make asynchronous NRQL queries that have longer query duration limits.
+---
+
+You can use our NerdGraph API to run an asynchronous NRQL query. This is useful for running queries that have less risk of being interrupted by browser or HTTP connection timeout limits or other issues. It's also especially applicable for running queries that may take a long time to complete.
+
+## Requirements [#requirements]
+
+The normal [NRQL query duration limit](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries) is one minute. To run queries with a 10-minute limit, you must have [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-plus). (edited) 
+
+## Asynchronous query example [#example]
+
+Here's an example of an asynchronous query NerdGraph call. The `async: true` is what makes the query asynchronous. This query will return results if it completes within the default timeout of 5 seconds; otherwise it will return `queryProgress` data for use in follow-up queries to the `actor.account.nrqlQueryProgress` field. 
```

Comment:
I'd add a sentence of exposition because when I got to the second example, I wasn't quite sure what exactly I was looking at until I reread it. Some ideas:

> An asynchronous query will return the results if it completes within the default timeout of 5 seconds. If an async query exceeds 5 seconds, then the query will return a `queryProgress` ID, which you can use to get results or the progress status of that query exceeding 5 seconds. 

It's a little redundant from a language-perspective, but I think being redundant works in the service of absolute clarity. A technical question I have is:

- Can you use the `queryProgress` ID to get partial results? i.e., if I had a stacked query to retrieve 2 discrete sets of data an datum A was retrieved before the code could get datum B, would I be able to use `queryProgress` to return A and get progress on B? Idk if that's just a science fiction example but it occurred to me~ 
- I'm not entirely clear what the `actor.account.nrqlQueryProgress` field is. Can you use `actor.account.nrqlQueryProgress` to do what I was asking about above ^?